app/views/page/newFlowHistoryPage.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
```

Comment:
It would help if you can point out which are the new files and which ones require a review.